### PR TITLE
Add descriptions to the variables in the minimal script

### PR DESCRIPTION
**Pull request by JesterOrNot**
Correct me if I'm wrong on any of these

### CODE REVIEW

File: install/aws-terraform/variables.tf [hunk 1]
```diff
@@ -4,16 +4,19 @@
...
+  description = "The AWS region that resources should be created in"
 }
 
 variable "domain" {
-  type = string
+  type        = string
+  description = "The domain that Gitpod should be installed on"
 }
 
 variable "chart_location" {
```
Comment (wulfthimm) [inline install/aws-terraform/variables.tf:22]: That is no good path declaration. Please use the `path.root` or `path.cwd` variable.

**Review by wulfthimm — CHANGES_REQUESTED** [review]

File: install/aws-terraform/variables.tf (hunk 1 above)
Comment (csweichel) [reply install/aws-terraform/variables.tf:22]: those variables are used in the script. However they cannot be used as default values in terraform

Comment (JesterOrNot) [reply install/aws-terraform/variables.tf:22]: He is referencing the chart path

**Review by wulfthimm — APPROVED** [review]
LGTM

**Pull request merged**